lms/djangoapps/certificates/admin.py

```diff
@@ -46,8 +47,16 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     list_display = ('description', '__unicode__')
 
 
+class GeneratedCertificateAdmin(admin.ModelAdmin):
+    """
+    Django admin customizations for GeneratedCertificae model
+    """
+    list_display = ('id', 'course_id', 'mode', 'user')
```

Comment:
Why just these fields of `GeneratedCertificate`?  Not that this should delay this PR merging, but I could imagine other fields being useful in this model.